sdk/schemaregistry/azure-data-schemaregistry-jsonschema/pom.xml

```diff
@@ -0,0 +1,143 @@
+<!--
+  ~ Copyright (c) Microsoft Corporation. All rights reserved.
+  ~ Licensed under the MIT License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-client-sdk-parent</artifactId>
+    <version>1.7.0</version> <!-- {x-version-update;com.azure:azure-client-sdk-parent;current} -->
+    <relativePath>../../parents/azure-client-sdk-parent</relativePath>
+  </parent>
+
+  <groupId>com.azure</groupId>
+  <artifactId>azure-data-schemaregistry-jsonschema</artifactId>
+  <version>1.0.0-beta.1</version> <!-- {x-version-update;com.azure:azure-data-schemaregistry-jsonschema;current} -->
+
+  <name>Microsoft Azure client library for Schema Registry JSON Schema Serializer</name>
+  <description>NetworkNT JSON Schema serializer for Azure Schema Registry client library</description>
```

Comment:
NetworkNT: Are we using [networknt/json-schema-validator](https://github.com/networknt/json-schema-validator), Is that a transitive dependency?

Reply:
Oops. Thanks for that catch. I'll remove it.